src/test/fcntl_pipe_sz.c

```diff
@@ -0,0 +1,33 @@
+/* -*- Mode: C; tab-width: 8; c-basic-offset: 2; indent-tabs-mode: nil; -*- */
+
+#include "rrutil.h"
+
+int main(void) {
+
+  int p[2];
+  int size;
+  int nsize;
+
+  test_assert(pipe(p) != -1);
+
+  size = fcntl(p[0], F_GETPIPE_SZ);
+
+  if (size == -1 && errno == EINVAL)
+  {
+    // should we succeed when PIPE_SZ is not supported?
```

Comment:
No. It's supported since 2.6.35 and rr definitely doesn't work at all before 3.6 or thereabouts, so these fcntls will definitely be supported.